typeDefs.js

```diff
@@ -197,10 +197,11 @@ const typeDefs = gql`
 
     "User Mutation"
     registerUser(user: UserInput!): User
-    loginUser(user: UserInput!): User
+    loginUser(user: UserInput!, language: Int!): User
     deleteUser(id: ID!): User
     updateUserById(user: UserInput!, id: ID!): User
     updateUserByToken(user: UserInput!): User
+    confirmUser(token: String!, language: Int!): Boolean
```

Comment:
Revert this change, please, we are going to send back unlocalized error messages and localize them only on backend

Reply:
removed